### DISCUSSION TITLE
Configure pack cache directory

### DISCUSSION THREAD

**Discussion by thepalbi** [github/securitylab#754] [General] [answered]
Hi team I'm running codeql inside a docker container internet isolated, and need to provide some downloaded qlpacks that are a dependencies of a lib of mine. Any ideas can I explicitly configure a path for the codeql package cache dir? If I use it locally, it's something like `$HOME/pablo/.codeql/packages...`

**Comment by aibaars** [comment, marked as answer]
The `--additional-packs` flag can be used to inform CodeQL about additional folders where QL packs can be found.

Instead of manually adding the `--additional-packs` flag each time you run CodeQL,  you can define default flags in a configuration file: https://docs.github.com/en/code-security/codeql-cli/using-the-codeql-cli/specifying-command-options-in-a-codeql-configuration-file

The default location of the configuration file is `$HOME/.config/codeql/config`, however you can override this by setting the `CODEQL_CONFIG_FILE` environment variable.
